tensorboards-web-app/rockcraft.yaml

```diff
@@ -0,0 +1,122 @@
+name: tensorboards-web-app
+summary: Tensorboards Web App
+description: |
+  This web app is responsible for allowing the user to manipulate Tensorboard
+  instances in their Kubeflow cluster. To achieve this it provides a user
+  friendly way to handle the lifecycle of Tensorboard CRs.
+version: v1.7.0_1
+license: Apache-2.0
+base: ubuntu:20.04
+
+services:
+  serve:
+    override: replace
+    summary: "tensorboards service"
+    command: gunicorn -w 3 --bind 0.0.0.0:5000 --access-logfile - entrypoint:app
+    startup: enabled
+    user: ubuntu
```

Comment:
Need to remove `user: ubuntu` in favour of new setup of non-root user.
It should be added outside of `services`:
https://github.com/canonical/seldonio-rocks/blob/main/seldon-core-operator/rockcraft.yaml#L17

Reply:
OK. My reference was the in-tree jupyter-web-app rock that still uses `user: ubuntu`..